Week1/home-work/queries.js

```diff
@@ -0,0 +1,95 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+   host: 'localhost',
+   user: 'hyfuser',
+   password: 'hyfpassword',
+   database: 'world'
+});
+
+connection.connect();
+
+function populationMoreThan20() {
+   var result = "SELECT * FROM countries WHERE countries.country_population >= 20000000";
+   connection.query(result, function (error, result) {
+      if (error) throw error;
+      console.log("countries with population greater than 20 million\n", result)
+   })
+};
+
+function withLand() {
+   var result = "SELECT * FROM countries WHERE countries.country_name LIKE '%land%'";
+   connection.query(result, function (error, result) {
+      if (error) throw error;
+      console.log("countries with the word land\n", result)
+   })
+};
+
+function between() {
+   var result = "SELECT country_name, country_population FROM countries WHERE countries.country_population BETWEEN 500000 AND 1000000";
+   connection.query(result, function (error, result) {
+      if (error) throw error;
+      else if (result == 0) console.log('No countries with population between 500,000 and 1 million')
+      else console.log(result);
+   })
+};
+
+
+function euroCountries() {
+   var result = "SELECT countries.country_name, countries.continent FROM countries WHERE countries.continent LIKE '%europe%'";
```

Comment:
%europe% is not entirely correct. '=' can be used to check equality. If there is a continent 'Teurope' by mistake, it will be matched.